test/timezone.mjs

```diff
@@ -417,6 +417,10 @@ describe('TimeZone', () => {
   });
   describe('getNextTransition works', () => {
     const nyc = Temporal.TimeZone.from('America/New_York');
+    // At the time of writing, the Kolkata TZ is not scheduled to have any
+    // transitions.
+    const kolkataTz = Temporal.TimeZone.from('Asia/Kolkata');
```

Comment:
May want to use `Etc/GMT+10` which is guaranteed to *never* have transitions so this test will never break due to India's govt decisions.

Reply:
Done.